src/components/SectionTile/SectionTile.tsx

```diff
@@ -0,0 +1,38 @@
+import { BoxOutline } from '@components/BoxOutline'
+import { FC, MouseEvent, ReactNode } from 'react'
+
+import styles from './SectionTile.module.scss'
+
+export interface SectionTileProps {
+  children: ReactNode
+  thumbnailUrl?: string
+  onClick?: (event: MouseEvent<HTMLDivElement>) => void
+  onDoubleClick?: (event: MouseEvent<HTMLDivElement>) => void
+  isActive?: boolean
+}
+
+export const SectionTile: FC<SectionTileProps> = ({
+  children,
+  thumbnailUrl = '',
+  onClick = () => {},
```

Comment:
Można się zastanowić czy jest sens dodawać te puste funkcje jak może być func / undefined to nie podając nic mamy undefined bez błędu